requests-for-comments/2022-06-01-session-analysis.md

```diff
@@ -0,0 +1,118 @@
+# Sessions in insights
+
+The goal of this document is to propose a specific approach to how we add session analysis to our insights product as defined in the [Product Context document](https://github.com/PostHog/product-internal/pull/307/files),
+
+For more details on the problem we're trying to solve and the customer needs, please reference that document.
+
+**Note:** The goal of this project is not to build a feature that allows users to find and explore specific sessions. Rather this feature is targeted at doing analysis on sessions in aggregate
+
+## Requirements
+
```

Comment:
We haven't defined what a "session" is which is key to success here.

- How is the "border" between sessions drawn? Is it a timeout, dependent on devices, etc?
- Does it only include events from the same library?
- Can sessions retroactively change if events are being backfilled?

cc @marcushyett-ph since this is as much of a product question than a technical one.

Reply:
Also kind of covered in the sped: https://github.com/PostHog/product-internal/pull/307/files